pkg/tests/testinfra/testinfra.go

```diff
@@ -288,6 +290,12 @@ func CreateGrafDir(t *testing.T, opts ...GrafanaOpts) (string, string) {
 			_, err = unifiedAlertingSection.NewKey("disabled_orgs", disableOrgStr)
 			require.NoError(t, err)
 		}
+		if !o.EnableLog {
+			logSection, err := cfg.NewSection("log")
```

Comment:
the `log` section is created [above](https://github.com/grafana/grafana/blob/649cd4bf7f8fa942e20bc0f2f1eeae7ffa5677db/pkg/tests/testinfra/testinfra.go#L180) so you can use `getOrCreateSection()` instead.